packages/mds-logger/loggers.ts

```diff
@@ -15,7 +15,7 @@ export const logger = (() =>
       messageKey: 'message',
       level: process.env.DEBUG ? 'debug' : false || 'info'
     },
-    pino.destination({ sync: false })
+    pino.destination({ sync: process.env.DEBUG ? true : false })
```

Comment:
Just a thought - could simplify this to:

```suggestion
    pino.destination({ sync: process.env.DEBUG })
```